firstvoices/backend/views/base_views.py

```diff
@@ -129,6 +131,8 @@ class SiteContentViewSetMixin:
     Provides common methods for handling site content, usually for data models that use the ``BaseSiteContentModel``.
     """
 
+    throttle_classes = [BurstRateThrottle, SustainedRateThrottle]
+
```

Comment:
Hmm. We actually have a couple of APIs that don't use either of these mixins (sites list, I think?). I don't want to have to add a new mixin to every API... pondering the best way to handle this.

Reply:
Maybe make a ThrottlingMixin and add that to the FVPermissions Mixin as well as our non-FVPermissions APIs? I don't think we have any APIs that use SiteContentMixin without FVPermissionsMixin, so could probably skip adding it in the SiteContent one.

Reply:
+1 to this solution